source/fundamentals/connection/network-compression.txt

```diff
@@ -0,0 +1,98 @@
+.. _compression:
+.. _network-compression:
+
+===================
+Network Compression
+===================
+
```

Comment:
is this and the Java page missing the front matter? e.g.
```suggestion
.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol
```

Reply:
Looks like it is... I will add it for Kotlin here at least